app/ide-desktop/lib/client/src/bin/server.ts

```diff
@@ -107,10 +108,28 @@ export class Server {
 
     /** Start the server. */
     run(): Promise<void> {
-        return new Promise((resolve, reject) => {
+        return new Promise(async (resolve, reject) => {
+            const defaultValidity = 365
+            const ca = await mkcert.createCA({
+                organization: 'Hello CA',
+                countryCode: 'NP',
+                state: 'Bagmati',
+                locality: 'Kathmandu',
```

Comment:
not sure we want dummy values like these, even if *almost* nobody is going to see them. maybe it's better to use the location of the US company - see the bottom of <https://enso.org/> for the address.

the same applies to the organization, not just the location